src/home_robot_hw/home_robot_hw/stretch_client/modules/manip.py

```diff
@@ -58,19 +58,18 @@ def get_joint_positions(self):
             q[HelloStretchIdx.ROLL],
         ]
 
-    @enforce_enabled
-    def wait(self):
-        self._ros_client.wait_for_trajectory_action()
-
     @enforce_enabled
     def goto(self, q, move_base=False, wait=True, max_wait_t=10.0, verbose=False):
```

Comment:
this is an aside but should we rename "goto"

Reply:
I was hoping to deprecate this function, but wanted to keep it for now so as to not break any existing code.